### PR TITLE
✨  Add edit cells

### PR DESCRIPTION
**Pull request by mholta**
- :bug: Specify dependency version number
- :package: Add new peer deps
- version packages
- version2
- move to components again
- ✨ Add react-hook-form edit cells and implement in example

## Issues related to this change:
[AB#348856](https://dev.azure.com/Equinor/5f972ff3-ed9a-4405-9db1-84542b5007a8/_workitems/edit/348856)

### CODE REVIEW

File: apps/jotai-form-example/src/features/unit-events-page/components/ActionsCell.tsx [hunk 1]
```diff
@@ -1,40 +1,78 @@
...
-  function handleCancel() {
-    resetForm()
+  const onSubmit = handleSubmit(
+    async (data) => {
+      try {
+        const newValues = removeFormMeta(data)
+
+        const isNew = getIsNew()
+        console.log(`Mutating ${isNew ? 'new' : 'updated'} Unit Event`, newValues)
+        await mutatePokemon([newValues])
```
Comment (mholta) [inline apps/jotai-form-example/src/features/unit-events-page/components/ActionsCell.tsx:42]: Mutate unit event

**Review by mholta — COMMENTED** [review]
Peer review with Constanza

File: packages/apollo-components/src/form-cells/EditableTextFieldCell.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import { TypographyCustom } from '@equinor/apollo-components'
 import { TextField } from '@equinor/eds-core-react'
 import { CellContext } from '@tanstack/react-table'
 import { Controller, FieldPath, useFormContext } from 'react-hook-form'
 import styled from 'styled-components'
+import { TypographyCustom } from '../cells'
```
Comment (consthauge) [inline packages/apollo-components/src/form-cells/EditableTextFieldCell.tsx:5]: We should have a discussion about accessibility

**Review by consthauge — CHANGES_REQUESTED** [review]

File: packages/apollo-components/src/form-cells/EditableTextFieldCell.tsx (hunk 2 above)
Comment (mholta) [reply packages/apollo-components/src/form-cells/EditableTextFieldCell.tsx:5]: Yes, but this PR should probably be merged before that discussion, as we have multiple projects depending on it

File: packages/apollo-components/src/cells/PopoverCell.tsx [hunk 3]
```diff
@@ -9,15 +9,19 @@ interface PopoverCellProps {
   title?: string | JSX.Element | ReactNode
 }
 
-export const PopoverCell = (props: PopoverCellProps) => {
+export function PopoverCell(props: PopoverCellProps) {
```
Comment (consthauge) [inline packages/apollo-components/src/cells/PopoverCell.tsx:12]: Har dere bestemt dere for å gå for denne type syntaks nå?

**Review by consthauge — APPROVED** [review]

File: packages/apollo-components/src/cells/PopoverCell.tsx (hunk 3 above)
Comment (mholta) [reply packages/apollo-components/src/cells/PopoverCell.tsx:12]: Yes :)

**Pull request merged**